personal_mycroft_backend/backend/precise.py

```diff
@@ -0,0 +1,24 @@
+from flask import request
+import time
+import json
+from os.path import join
+from personal_mycroft_backend.backend.decorators import noindex
+from personal_mycroft_backend.settings import PRECISE_DATA_FOLDER
+
+
+def get_precise_routes(app):
+    @app.route('/precise/upload', methods=['POST'])
+    @noindex
+    def precise_upload():
+        uploads = request.files
+        for precisefile in uploads:
+            fn = uploads[precisefile].filename
+            if fn == 'audio':
+                name = (str(int(time.time()))) + ".wav"
+                uploads[precisefile].save(os.path.join(app.config['UPLOAD_FOLDER'], name))
```

Comment:
shouldn't  ```app.config['UPLOAD_FOLDER']  ```be ``` PRECISE_DATA_FOLDER``` ?

Reply:
Probably, this was a copy and paste from my repo.